src/components/withAnimatedRef.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {useAnimatedRef} from 'react-native-reanimated';
+import PropTypes from 'prop-types';
+import getComponentDisplayName from '../libs/getComponentDisplayName';
+
+export default function withAnimatedRef(WrappedComponent) {
+    function WithAnimatedRef(props) {
+        const animatedRef = useAnimatedRef();
+        return (
+            <WrappedComponent
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...props}
+                ref={props.forwardedRef}
+                animatedRef={animatedRef}
+            />
+        );
+    }
+    WithAnimatedRef.displayName = `withAnimatedRef(${getComponentDisplayName(WrappedComponent)})`;
+    WithAnimatedRef.propTypes = {
```

Comment:
@robertKozik we have a console error on web.
Steps
1. `npm run web`
2. Login


`Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?`


<img width="594" alt="image" src="https://github.com/Expensify/App/assets/29673073/8b0162e6-9d80-448a-8b24-a4001bec00c4">